akismet-privacy-policies.php

```diff
@@ -187,14 +250,13 @@ public function error_message() {
 			return NULL;
 		}
 
-		$options = get_option( 'akismet_privacy_notice_settings' );
-		if ( empty( $options[ 'error_message' ] ) ) {
-			$options[ 'error_message' ] = $this->error_message;
+		if ( empty( $this->options[ 'error_message' ] ) ) {
+			$this->options[ 'error_message' ] = $this->error_message;
 		}
 
 		// check for checkbox active
 		if ( isset( $_POST[ 'comment' ] ) && ( ! isset( $_POST[ 'akismet_privacy_check' ] ) ) ) {
-			$message = apply_filters( 'akismet_privacy_error_message', $options[ 'error_message' ] );
+			$message = apply_filters( 'akismet_privacy_error_message', __( $this->options[ 'error_message' ], 'akismet-privacy-policies' ) );
```

Comment:
Need escaping.

Reply:
Done

> Also please add a note about this new feature, change inside the CHANGELOG file. Thx heart @nuss

Oh! I think you've added the changelog after I had forked the repo, so it's missing in my fork. I'll figuere it out... Thanks very much!